### ISSUE TITLE
Support for seafile server / client 4.0.6

### ISSUE THREAD

**Issue by mdriesch** [szimszon/tb-filelink-seafile#6]
Thanks for your work. I get the following error message:

> Unable to upload file.jpg to SeaFile.

Any ideas about that? Is it a version issue?
Kind regards,
Michael

**Comment by szimszon** [comment]
Are there any firewall that could block http traffic to seafile server?

I can't reproduce the failure. :(

**Comment by mdriesch** [comment]
No. It's all 'local', so firewall should not be an issue. I use https though. I read somewhere, that it might be an issue with the certificate. If SF 4.0.6 is working for you (?), then i will fokus on that possibility. Many thanks, Michael

Am 9. März 2015 21:37:18 MEZ, schrieb szimszon <notifications@github.com>:
>Are there any firewall that could block http traffic to seafile server?
>
>I can't reproduce the failure. :(
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/szimszon/tb-filelink-seafile/issues/6#issuecomment-77936833

-- 
Diese Nachricht wurde von meinem Android-Mobiltelefon mit K-9 Mail gesendet.

**Comment by szimszon** [comment]
Yes, definitively works for me.

You have to have a demoCA and import the root CA into the thunderbird or you can request a free cert from startssl.com
